RELEASE_NOTES.md

```diff
@@ -1,5 +1,10 @@
 # Git Town Release Notes
 
+## Unreleased
+
+* `git sync --all`: push tags
```

Comment:
Also lets people know whats already in it in the case that we don't release for a while. Though hopefully this serves as a reminder to release often